spec/abstract/ARM_HYP/ArchVSpace_A.thy

```diff
@@ -537,7 +537,7 @@ where
           gicIndices \<leftarrow> return [0..<num_list_regs];
 
           mapM (\<lambda>vreg. do
-                    val \<leftarrow> do_machine_op $ get_gic_vcpu_ctrl_lr (of_int vreg);
+                    val \<leftarrow> do_machine_op $ get_gic_vcpu_ctrl_lr (of_nat vreg);
```

Comment:
`upt` always produced `nat`s, so I guess the type of `of_int` became a bit less permissive?

Reply:
Yes, of-int changed upstream and now actually wants an int. There were one or two more instances of this somewhere.